src/Confirm/Confirm.js

```diff
@@ -95,3 +95,5 @@ Confirm.propTypes = {
   rightButtonClassName: PropTypes.string,
   rightButtonCallback: PropTypes.func.isRequired
 };
+
+module.exports = Confirm;
```

Comment:
Have to do this now when exporting. Apparently Babel 5 was doing it wrong and they've changed the way that `export default` works, so this is why there's many files that have this change.